core/src/main/scala/io/finch/syntax/scala/ScalaToTwitterFuture.scala

```diff
@@ -1,21 +1,30 @@
 package io.finch.syntax.scala
 
-import scala.concurrent.{ExecutionContext, Future => ScalaFuture}
+import scala.concurrent.{Future => ScalaFuture}
 import scala.util.{Failure, Success}
 
 import com.twitter.util.{Future, Promise}
 import io.finch.syntax.ToTwitterFuture
 
 trait ScalaToTwitterFuture {
-  implicit def scalaToTwitterFuture(implicit ec: ExecutionContext): ToTwitterFuture[ScalaFuture] =
+  implicit lazy val scalaToTwitterFuture: ToTwitterFuture[ScalaFuture] =
```

Comment:
Just make it a `val` (no need for `lazy`).

Reply:
Done.